netlogo-gui/src/main/properties/ReporterLineEditor.scala

```diff
@@ -6,7 +6,7 @@ package properties
 import org.nlogo.awt.Fonts.platformMonospacedFont
 import javax.swing.{JScrollPane, ScrollPaneConstants}
 import editor.{EditorField, Colorizer}
-import java.awt.{Dimension, Font}
+import java.awt.{Dimension, Font, GridBagConstraints}
```

Comment:
put before javax import
you could move the org.nlogo import below the editor one